crates/host-sys/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+use wasmcloud_host::wasmbus::{Host, HostConfig};
+
+// TODO: Proper error handling
+
+#[no_mangle]
+pub extern "C" fn run_host() {
+    tokio::runtime::Builder::new_multi_thread()
+        .enable_time()
+        .enable_io()
```

Comment:
`.enable_all()` would do both here

Reply:
Only concern there would be the doc comment
```
If additional components are added to Tokio in the future, enable_all will include these future components.
```

That also would mean we get additional components without changing the code, depends on what we'd rather have

Reply:
yeah that's definitely true -- if a breaking change were added to `tokio` this might need to be changed when upgrading the dependency could be extra work to do/needing to be careful around the upstream releases

Reply:
I just switched to https://docs.rs/tokio/latest/tokio/runtime/struct.Runtime.html#method.new, which should enable everything we need here